react/features/recent-list/components/AbstractRecentList.js

```diff
@@ -1,62 +0,0 @@
-// @flow
```

Comment:
Nothing really, I just followed the coding style of other components, but then I found out that there likely won't be a web version ever of these lists. What do you think?

Reply:
I think that's okay. It's not that hard to create an Abstract. It might be better create Abstracts as needed anyway, to avoid doing additional work that doesn't get used. This assumes the concrete version still attempts to break apart logic into separate functions.